packages/polar-betterauth/src/plugins/checkout.ts

```diff
@@ -47,6 +47,13 @@ export const checkout =
 								z.union([z.string(), z.number(), z.boolean()]),
 							)
 							.optional(),
+						allowDiscountCodes: z
+							.string()
+							.toLowerCase()
+							.transform((x) => x === "true")
+							.pipe(z.boolean())
+							.optional(),
```

Comment:
this could be improved to be `z.coerce.boolean().optional().default(true)`

Reply:
Fixed!